### PR TITLE
fix-source-false

### PR DESCRIPTION
**Pull request by tarunsingh5**
fix raag source showing as 'false' for dasam bani

### CODE REVIEW

**Comment by tarunsingh5** [comment]
#216 #214 fixed by this

File: src/js/search.js [hunk 1]
```diff
@@ -70,6 +70,11 @@ function getShabadHyperLink (shabad) {
   return `/shabad?id=${shabad.shabadid}&q=${q}${type ? `&type=${type}` : ''}${source ? `&source=${source}` : ''}`;
 }
 
+function getRaagOrNull(Raag) {
+  if(Raag == "No Raag" ||  Raag == null){ return ""}
```
Comment (bogas04) [inline src/js/search.js]: Could we please have it as `raag` since it's not a class per-se?

Comment (bogas04) [reply src/js/search.js]: Also, there are some linting errors. Try to run `npm run test`. To find those.

**Comment by tarunsingh5** [comment]
i changed Raag to raag and removed linting errors from my changes

**Comment by navdeepsinghkhalsa** [comment]
Fixes implemented

**Pull request merged**